templates/plugin/rabbitmq.conf.erb

```diff
@@ -0,0 +1,19 @@
+# TODO need to find a better way to handle the types.db config
+TypesDB "/usr/share/collectd/types.db"
+TypesDB "/usr/share/collect-rabbitmq/types.db.custom"
+
+LoadPlugin python
+<Plugin python>
```

Comment:
This seems remarkably similar to the way that genericjmx is written. Can you refactor around the way the python plugin is written?